### PR TITLE
[fix bug 1147541] Fix 500 error when using an invalid username.

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

File: remo/base/settings_urls.py [hunk 1]
```diff
@@ -4,6 +4,6 @@
 urlpatterns = patterns(
     '',
     url(r'^$', 'remo.base.views.edit_settings', name='edit_settings'),
-    url(r'^availability/(?P<display_name>[A-Za-z0-9_]+)/$',
+    url(r'^availability/(?P<display_name>[\w._\-]+)/$',
```
Comment (johngian) [inline remo/base/settings_urls.py]: ``\w`` includes ``_``.

File: remo/profiles/models.py [hunk 2]
```diff
@@ -110,9 +110,9 @@ class UserProfile(caching.base.CachingMixin, models.Model):
         max_length=DISPLAY_NAME_MAX_LENGTH, blank=True, default='',
         unique=True,
         validators=[
-            RegexValidator(regex=r'("")|(^[A-Za-z0-9_]+$)',
-                           message='Please only A-Z characters, numbers and '
-                                   'underscores.')])
+            RegexValidator(regex=r'^[\w._\-]+$',
```
Comment (johngian) [inline remo/profiles/models.py]: ``.`` allows any character you might need to escape it with ``\.``

Comment (johngian) [inline remo/profiles/models.py]: ``.`` allows any character you might need to escape it with ``\.``

Comment (akatsoulas) [inline remo/profiles/models.py]: This is not true if `.` is inside `[]`
https://docs.python.org/2/library/re.html

**Comment by glogiotatidis** [comment]
update comment per irc chat r+wc

**Pull request merged**